pkg/deploy/cmd/delete_test.go

```diff
@@ -221,13 +211,12 @@ func TestStop(t *testing.T) {
 				ktestclient.NewGetAction("deploymentconfigs", "default", "config"),
 			},
 			kexpected: []ktestclient.Action{
-				ktestclient.NewListAction("replicationcontrollers", "default", kapi.ListOptions{}),
+				ktestclient.NewListAction("replicationcontrollers", "default", kapi.ListOptions{LabelSelector: labels.SelectorFromSet(map[string]string{"openshift.io/deployment-config.name": "config"})}),
```

Comment:
no, this test should smack us if we change the constant's value and break people